### PR TITLE
Update PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by MickStanciu**
add a new checkbox for breaking changes

## Summary

new check-box for breaking changes

## Checklist

- [x] Is this PR a reasonable size?

- [ ] List deployment sequence with all relevant PRs.
  - ...

---

**Code Review Guidelines** for Reviewers

- Try to review in a timely manner. Opinions/nitpicks should not be blockers. Pair on a call for non-trivial feedback.
- Overall design and approach should follow established patterns. Don't try to make the PR perfect.
- Try to identify edge cases, race conditions, over-engineering, lack of test coverage and complexity.
- If you don't feel qualified to review the code, pass it on to someone who is.

### CODE REVIEW

**Review by R-cen — DISMISSED** [review]

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -20,6 +20,7 @@
 
 - [ ] List deployment sequence with all relevant PRs.
   - ...
+- [ ] If you have breaking changes, tick this to confirm that you checked other repositories in order to make sure they are not affected
```
Comment (lachlancooper) [inline PULL_REQUEST_TEMPLATE.md:23]: What if other repositories _are_ affected? I think we could be more opinionated and provide more detailed advice here.
```suggestion

<!-- Does this PR cause breaking changes? If so, it is your responsibility to check that other repositories are not affected. If they are, you must plan a deployment sequence (see above) to ensure any impact is addressed. Once you have done this, put an `x` in the box -->

- [ ] For breaking changes: I have confirmed that other repos are not affected, or I have planned a deployment sequence to address any impact.
```

**Review by lachlancooper — COMMENTED** [review]
This was my suggestion so I'm going to leave this for @R-cen to review.

**Review by R-cen — APPROVED** [review]
